utils/custom_errors_test.go

```diff
@@ -0,0 +1,31 @@
+package utils_test
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/smartcontractkit/chainlink/utils"
+)
+
+func TestUtils_StatusCodeForError_ValidationError(t *testing.T) {
+	t.Parallel()
+	expectedStatusCode := 400
+	actualStatusCode := utils.StatusCodeForError(utils.NewValidationError("test"))
+	assert.Equal(t, actualStatusCode, expectedStatusCode)
+}
+
+func TestUtils_StatusCodeForError_DatabaseAccessError(t *testing.T) {
+	t.Parallel()
+	expectedStatusCode := 500
+	actualStatusCode := utils.StatusCodeForError(utils.NewDatabaseAccessError("test"))
+	assert.Equal(t, actualStatusCode, expectedStatusCode)
+}
+
+func TestUtils_StatusCodeForError_DefaultError(t *testing.T) {
+	t.Parallel()
```

Comment:
Can we convert these three tests into one table driven test ala https://github.com/smartcontractkit/chainlink/blob/master/utils/utils_test.go#L76-L98?